types/cesium/index.d.ts

```diff
@@ -121,11 +121,14 @@ declare namespace Cesium {
         readonly isConstant: boolean;
         propertyNames: string[];
 
+        constructor(value: object, createPropertyCallback?: (value?: any) => void);
```

Comment:
Based on https://cesium.com/docs/cesiumjs-ref-doc/PropertyBag.html?classFilter=propertybag#PropertyBag, value is also optional

Reply:
thanks @jrafidi